Doc/library/py_compile.rst

```diff
@@ -40,7 +40,8 @@ byte-code cache files in the directory containing the source code.
    when an error is encountered while compiling *file*. If *doraise* is false
    (the default), an error string is written to ``sys.stderr``, but no exception
    is raised.  This function returns the path to byte-compiled file, i.e.
-   whatever *cfile* value was used.
+   whatever *cfile* value was used. If *quite* is False or 0, a full output will
+   be returned, onlly errors when 1 and nothing when 2.
```

Comment:
The _quiet_ mode should not affect the function’s return value. And there is no difference between “full output” and “only errors”. Based on your implementation, I would move the existing text for _doraise_ to a new paragraph, something like:

The _doraise_ and _quiet_ arguments determine how errors are handled while compiling *file*. If _quiet_ is 0 or 1, and *doraise* is false, the default behaviour is enabled: an error string is written to ``sys.stderr``, and the function returns None instead of a path. If *doraise* is true, a ``:exc:`PyCompileError` `` is raised instead. However if _quiet_ is 2, no message is written, and _doraise_ has no effect.